nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -715,6 +785,40 @@ public static Nullness getGenericReturnNullnessAtInvocation(
       MethodInvocationTree tree,
       VisitorState state,
       Config config) {
+
+    List<? extends Tree> typeArgumentTrees = tree.getTypeArguments();
```

Comment:
Does this new logic only apply to generic methods?  If so, can we only run it for that case, by checking if the invoked method has any type variables, `!invokedMethodSymbol.getTypeParameters().isEmpty()`?